src/abstractdataframe/prettytables.jl

```diff
@@ -0,0 +1,62 @@
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+#
+# Description
+# ==============================================================================
+#
+#     This file contains the definitions related to the interface with
+#     PrettyTables.jl
+#
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+# Default DataFrames highlighter for text backend.
+#
+# This highlighter change the text color to gray in cells with `nothing`,
+# `missing`, `#undef`, and types related to DataFrames.jl.
+function _df_h_f(data,i,j)
+    try
+        return ismissing(data[i,j]) ||
+            data[i,j] == nothing ||
```

Comment:
```suggestion
            isnothing(data[i,j]) ||
```

Reply:
Can we use `isnothing`?  I think it is not supported in Julia 1.0.

Reply:
right, then use `===`.